src/modules/loggers/tealium.ts

```diff
@@ -1,16 +1,10 @@
-import { ITealium } from '../../model/tealium';
+import { ITealiumV1, ITealiumV2 } from '../../model/tealium';
 import {excludeLogs, sendEventToFlipper} from '../init';
 
-export const createTealiumLogger = (Tealium: any, config: ITealium, printLogs: boolean = false) => {
-  Tealium.initialize(
-    config.account,
-    config.profile,
-    config.environment,
-    config.iosDatasource,
-    config.androidDatasource,
-    config.instance,
-    config.isLifecycleEnabled,
-  );
+export const createTealiumLogger = (Tealium: any, config: ITealiumV1 | ITealiumV2, printLogs: boolean = false) => {
+  Tealium.initialize({
```

Comment:
https://docs.tealium.com/platforms/react-native-v1/install/#initialize

It will not work with v1 of Tealium.
Please update the readme + package.json peerDependencies to upgrade the supported version to v2